### ISSUE TITLE
Gs0131983 - Peatland microbial communities from Stordalen Mire, Sweden

### ISSUE THREAD

**Issue by aclum** [microbiomedata/issues#936]
157 metatranscriptomes
662 metagenomes

jgi proposal ID: 503530
JGI proposal title: Illuminating the pathways to carbon liberation: a systems and modeling approach to resolving the 'consequential unknowns' of carbon transformation and loss from thawing permafrost peatlands
GOLD study ID: Gs0131983
PI: Saleska
co-PI: Rich
BER Support Science

TODO: Check on env triad terms

should be under EMERGE consortium

Try to bring this in via a submission portal entry.

**Comment by aclum** [comment]
@mslarae13  can you confirm there was an email sent for this?

**Comment by mslarae13** [comment]
I missed this one. I'll get the email draft written and sent to Emiley by Monday

**Comment by aclum** [comment]
@mslarae13 @emileyfadrosh did this email get sent?

**Comment by ssarrafan** [comment]
Leaving this in the sprint till Friday since @mslarae13 is working this week.  Can you please update this issue?

**Comment by mslarae13** [comment]
@emileyfadrosh I sent you the draft email on 12/16. Can you send it to the PI when the holiday ends?

**Comment by aclum** [comment]
@emileyfadrosh @mslarae13 i believe this is still outstanding.

**Comment by ssarrafan** [comment]
@aclum @mslarae13 should this go to the backlog or stay in the active sprint?

**Comment by aclum** [comment]
This should go in this sprint, we are planning on reaching out to their data management person this week.

**Comment by aclum** [comment]
We are meeting and working with their metadata person.

**Comment by aclum** [comment]
Check in after Feb 10 if we don't hear from them sooner.

**Comment by ssarrafan** [comment]
> Check in after Feb 10 if we don't hear from them sooner.

@aclum just a reminder that it's past Feb 10th

**Comment by mslarae13** [comment]
@aclum status on this?? I don't _think_ I've received a submission for this?

**Comment by aclum** [comment]
@mslarae13  no, we need to meet with them. At this point we should plan do to this after the renewal.
